src/main/java/seedu/address/model/module/UniqueModuleList.java

```diff
@@ -24,6 +24,7 @@
     private final ObservableList<Module> internalList = FXCollections.observableArrayList();
     private final ObservableList<Module> internalUnmodifiableList =
             FXCollections.unmodifiableObservableList(internalList);
+    private int mcCount = 0;
```

Comment:
Is it possible to leave MC count outside of this class? Because I was thinking this class might be used for stuff like a list of prerequisites for a module, etc, where MC count wouldn't really be relevant?

Reply:
Will create a new class that extends to include MC Count.